api/scpca_portal/common.py

```diff
@@ -18,6 +18,9 @@
 
 TEMPLATE_PATH = CODE_PATH / "scpca_portal" / "templates"
 
+TEST_INPUT_BUCKET_NAME = "scpca-portal-public-test-inputs/2024-04-19/"
+INPUT_BUCKET_NAME = TEST_INPUT_BUCKET_NAME if settings.TEST else "scpca-portal-inputs"
```

Comment:
The output bucket is in `config/local.py` and/or `config/production.py`. Should we move the input bucket/s to the `config/` dir as well?